.github/workflows/e2e-embed-react.yml

```
@@ -26,6 +26,7 @@ jobs:
       - uses: ./.github/actions/cache-build
       - name: Run Tests
         run: yarn test-e2e:embed-react
+        run: yarn workspace @calcom/embed-react packaged:tests
```

This is invalid syntax

```suggestion
        run: |
          yarn test-e2e:embed-react
          yarn workspace @calcom/embed-react packaged:tests
```

---

Is it invalid syntax or not a recommended approach? Syntax wise it's working already.